types/crack-json/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "crack-json" {
+    function extractJson(
+        subject: string,
+        configuration?: {
+            filter?: (input: string) => boolean;
+            parser?: (input: string) => any;
+        },
+    ): any;
```

Comment:
The library is essentially just a wrapper around `JSON.parse`, which is also typed as returning `any`, so I suppose `any` is okay to use here

Reply:
export named function:
```ts
export function extractJson(....): any;
```
the module exports just named function (and eventually types for configuration object)

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html

Reply:
Thanks. Turns out my local install wasn't working right, thats why I couldn't get it to work wihout the `declare module`